src/components/Calculator.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './claculator.css';
+
+const Calculator = () => (
```

Comment:
- [ ] Same remark about class-based, the `Calculator.js` component should be class-based, please kindly change it 🙃